src/Nox.Generator/Application/Commands/CreateCommandGenerator.cs

```diff
@@ -25,10 +25,14 @@ public void Generate(SourceProductionContext context, NoxSolutionCodeGeneratorSt
 
             var primaryKeysQuery = string.Join(", ", entity.Keys.Select(k => $"entityToCreate.{k.Name}.Value"));
 
+            //TestEntityTwoRelationshipsOneToOne has more than one relationship to SecondTestEntityTwoRelationshipsOneToOne
```

Comment:
we do not want this comments on the template?

Reply:
This a comment for your attention actually, we calculate the related entities here with group by method.